app/helpers/flash_messages_helper.rb

```diff
@@ -17,7 +17,7 @@ def bulma_class_for(flash_type)
   def flash_messages
     flash.map do |msg_type, message|
       content_tag(:div, class: "notification #{bulma_class_for(msg_type)}") do
-        content_tag(:button, "", class: "delete") + message
```

Comment:
There was no behaviour behind it.
Might change it back somewhere in to future, but for now its ok to me removed